events/battle_events.txt

```diff
@@ -2437,38 +2448,88 @@ character_event = {
 		}
 		add_trait = desert_terrain_leader
 	}
+	#option = {
```

Comment:
Starting from here. Why it's commented out? These are the same options. Only triggers should be modified.